core/node/eth_sender/src/tester.rs

```diff
@@ -406,15 +411,19 @@ impl EthSenderTester {
 
     pub async fn save_execute_tx(&mut self, l1_batch_number: L1BatchNumber) -> EthTx {
         assert_eq!(l1_batch_number, self.next_l1_batch_number_to_execute);
+        let l1_batch_headers = vec![
+            self.get_l1_batch_header_from_db(self.next_l1_batch_number_to_execute)
+                .await,
+        ];
         let operation = AggregatedOperation::Execute(ExecuteBatches {
-            l1_batches: vec![
-                self.get_l1_batch_header_from_db(self.next_l1_batch_number_to_execute)
-                    .await,
-            ]
-            .into_iter()
-            .map(l1_batch_with_metadata)
-            .collect(),
-            priority_ops_proofs: Vec::new(),
+            priority_ops_proofs: l1_batch_headers
+                .iter()
+                .map(|_| Default::default())
+                .collect(),
```

Comment:
Nit: Can't this be expressed as `vec![Default::default(); l1_batch_headers.len()]`?